### ISSUE TITLE
Plugin Dead?

### ISSUE THREAD

**Issue by Lorccan** [naofireblade/homebridge-neato#79]
Is there any realistic prospect of an update to this plugin?

I notice the author’s reply in 2021 that it was not dead, but that s/he had little time, but it’s 2023 and … nothing.

**Comment by naofireblade** [comment]
Yes! I'm working on a rebuild with dynamic platform to bring more stability and no more disappearing robots after neato server outages: https://github.com/naofireblade/homebridge-neato/commits/dynamic-platform

**Comment by Lorccan** [comment]
I assume that this never went anywhere?
